.github/workflows/rotki_nightly.yml

```
@@ -58,7 +35,7 @@ jobs:
           pip install codecov pytest-cov
           pip install -r requirements_dev.txt
           pip install -e .
-      - name: Run Test
+      - name: Run Tests
```

for consistency with what you did in the other file
```suggestion
      - name: Run tests
```